### DISCUSSION TITLE
Union, Intersection, Complement

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#492] [Ideas]
Hi,

I'm coming to meow after using Vis for a number of years.

Vis has these neat operators for acting on marks by selecting the union, intersection, and complement of the marks.

See here:

https://github.com/martanne/vis/wiki/Differences-from-Kakoune

Any thoughts on how to get this functionality in meow?

**Comment by DogLooksGood** [comment]
Sorry, I have no experience with vis. Do you have any examples, or what you can achieve by using these functionalities?
